settings_dialog.js

```diff
@@ -1,41 +1,55 @@
 // Everything under the settings button
 "use strict";
 
+const fs = require('fs-extra');
 const path = require('path');
 
 var {shell} = require('electron');
+const {dialog} = require('electron').remote;
+const win = remote.getCurrentWindow();
+
+const {getVersionData} = require("./version_info.js");
 
 const { Modal, showGenericError} = require('./modal');
-const {listInstalledVersions, deleteInstalledVersion} = require('./install_handler.js');
-const { settings, saveSettings, installPath } = require('./settings.js');
+const { listInstalledVersions, deleteInstalledVersion } = require('./install_handler.js');
+
+const { settings, dataFolder, saveSettings, defaultInstallPath } = require('./settings.js');
 
 const settingsModal = new Modal("settingsModal", "settingsModalDialog", {
     closeButton: "settingsClose"
 });
 
+const movingFileModal = new Modal("movingFileModal", "movingFileModalDialog", {
+    autoClose: false
+});
+
 // Used to skip callbacks on loading settings
 let loadingSettings = false;
 
+let listInstalledVersionsError = false;
```

Comment:
This variable is not read anywhere (used for anything) and can be removed.

Reply:
I agree with Hhyyrylainen here. The variable seems redundant in its current state, and as it is just creates unneeded noise in the project. I say either make use of it, or strip it out.

Reply:
True, that will be removed.

Reply:
True, that will be removed.